ruby/ql/lib/codeql/ruby/ast/Module.qll

```diff
@@ -14,6 +14,9 @@ class Module extends TModule {
   /** Gets the super class of this module, if any. */
   Module getSuperClass() { result = getSuperClass(this) }
 
+  /** Gets a sub class of this module, if any. */
+  Module getASubClass() { this = getSuperClass(result) }
```

Comment:
This predicate returns direct sub classes only, doesn't it? Let document that and perhaps also make it more clear from its name.